index.html

```diff
@@ -2,89 +2,90 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <title>My Basic Site
+  <title>Brannon Crumpton</title>
 
-  Leave these links here...
-  But this should be in a comment...
+  <!-- Leave these links here...
+  But this should be in a comment... -->
   <link href="https://fonts.googleapis.com/css?family=Cormorant|Poppins:300,400" rel="stylesheet">
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="styles/style.css">
 </head>
 <body>
-  <div class="bender">
-    <nav class="bender__nav">
-      <div class="box">
-        <div class="abc">
-          <h1 class="abc__first">Shia LaBeouf</h1>
-
-          <div class="abc__second">
-            <ul class="fry">
-              <li class="fry__thing">
-                <a class="fry__zelda" href="#welcome">Welcome</a>
-
-              <li class="fry__thing">
-                <a class="fry__zelda" href="#about">About</a>
+  <div class="app">
+    <nav class="app__nav">
+      <div class="container">
+        <div class="top-nav">
+          <h1 class="top-nav__left">Shia LaBeouf</h1>
+
+          <div class="top-nav">
```

Comment:
Watch out, here the old class was `abc__second`, you replaced `abc` with `top-nav` but see if there is a replacement for `__second`.